src/invidious/views/watch.ecr

```diff
@@ -140,7 +140,7 @@ we're going to need to do it here in order to allow for translations.
             <% if user %>
                 <% playlists = PG_DB.query_all("SELECT id,title FROM playlists WHERE author = $1 AND id LIKE 'IV%'", user.email, as: {String, String}) %>
                 <% if !playlists.empty? %>
-                    <form data-onsubmit="return_false" class="pure-form pure-form-stacked" action="/playlist_ajax" method="post">
+                    <form data-onsubmit="return_false" class="pure-form pure-form-stacked" action="/playlist_ajax" method="post" target="_blank">
```

Comment:
Is it really necessary to open a new tab for this form? Should the endpoint return a redirect instead?

Reply:
This is because the front-end has to work without Javascript. So, in order to not do a useless page refresh (which could update the related videos in the right sidebar, e.g) it has to be done in a new window/tab.